### PR TITLE
Numerous improvements to AutoVJ

### PR DESCRIPTION
**Pull request by worldveil**
- New .lxp project file
- Improved handling of repeated phrases
- Colorizing UP and DOWN patterns to be palette-compatible
- Better choices of patterns for CHORUS, UP, DOWN

### CODE REVIEW

File: .lxpreferences [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "version": "0.4.1",
-  "projectFileName": "Vehicle.lxp",
+  "projectFileName": "AutoVJ.lxp",
```
Comment (raldi) [inline .lxpreferences:3]: Is this intentional?

File: Projects/AutoVJ_old.lxp [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
```
Comment (raldi) [inline Projects/AutoVJ_old.lxp:1]: Just delete this if it's not going to be used anymore.

File: src/main/java/titanicsend/app/TEAutopilot.java [hunk 3]
```diff
@@ -22,9 +22,11 @@ public class TEAutopilot implements LXLoopTask {
     private boolean enabled = false;
     // should we try to sync BPM when ProDJlink seems off?
     private boolean autoBpmSyncEnabled = false;
+    private double TEMPO_DIFF_THRESHOLD = 0.05;
```
Comment (raldi) [inline src/main/java/titanicsend/app/TEAutopilot.java:25]: Needs comment

**Review by raldi — COMMENTED** [review]
Just a few cleanups/questions

File: .lxpreferences (hunk 1 above)
Comment (worldveil) [reply .lxpreferences:3]: no, sigh haha. the UI always does this and I always have to go back before I merge and change it

File: src/main/java/titanicsend/app/TEAutopilot.java (hunk 3 above)
Comment (worldveil) [reply src/main/java/titanicsend/app/TEAutopilot.java:25]: will do!

**Pull request merged**